File: java/Program/Petshop.java
public class Petshop {
    private int id;
    private String name;
    private int stock;
    private double price;

    public Petshop() {
        this.id = 0;
        this.name = "";
        this.stock = 0;
        this.price = 0.0;
    }

    public Petshop(int id, String name, int stock, double price) {
        this.id = id;
        this.name = name;
        this.stock = stock;
        this.price = price;
    }

    public void setName(String name) { this.name = name; }
    public void setId(int id) { this.id = id; }
    public void setStock(int stock) { this.stock = stock; }
    public void setPrice(double price) { this.price = price; }

    public int getId() { return this.id; }
    public String getName() { return this.name; }
    public int getStock() { return this.stock; }
    public double getPrice() { return this.price; }
}

File: php/Program/Petshop.php
<?php
class Petshop {
    protected $id;
    protected $name;
    protected $stock;
    protected $price;
    protected $photo;

    public function __construct($id = 0, $name = "", $stock = 0, $price = 0.0, $photo = "") {
        $this->id = $id;
        $this->name = $name;
        $this->stock = $stock;
        $this->price = $price;
        $this->photo = $photo;
    }

    public function setId($id) { $this->id = $id; }
    public function getId() { return $this->id; }
    
    public function setName($name) { $this->name = $name; }
    public function getName() { return $this->name; }
    
    public function setStock($stock) { $this->stock = $stock; }
    public function getStock() { return $this->stock; }
    
    public function setPrice($price) { $this->price = $price; }
    public function getPrice() { return $this->price; }
    
    public function setPhoto($photo) { $this->photo = $photo; }
    public function getPhoto() { return $this->photo; }
}